docs/resources/share.md

```diff
@@ -75,7 +75,7 @@ The following arguments are required:
 ### object Configuration Block
 
 * `name` (Required) - Full name of the object, e.g. `catalog.schema.name` for a table.
-* `data_object_type` (Required) - Type of the object, currently only `TABLE` is allowed.
+* `data_object_type` (Required) - Type of the data object.
```

Comment:
Could this `data object` be rewritten as the list of accepted values please? It would be super handy for the users of the module. Thanks!

Reply:
Added supported data object types.
`data_object_type` (Required) - Type of the data object, currently `TABLE`, `SCHEMA`, `VOLUME`, `NOTEBOOK_FILE` are supported.